WordPress/src/main/java/org/wordpress/android/ui/people/PeopleManagementActivity.java

```diff
@@ -55,7 +56,10 @@ public void onCreate(Bundle savedInstanceState) {
         }
 
         if (savedInstanceState == null) {
-            PeopleTable.deletePeopleForLocalBlogIdExceptForFirstPage(blog.getLocalTableBlogId());
+            // only delete cached people if there is a connection
+            if (NetworkUtils.isNetworkAvailable(this)) {
```

Comment:
So, this is kind of what I was asking in my comment [here](https://github.com/wordpress-mobile/WordPress-Android/pull/4120#issuecomment-221888645). However, I don't agree with showing any messages here. We'll be trying to fetch a fresh list of users and if network connection is not available we already show a message there.

Since this is mostly a hack to make sure removed users in remote are removed in our cache as well, I wouldn't do anything UI related here and will defer to the fetching method to inform the user.